### PR TITLE
Finished commenting and adding in column names from separate class

### PR DESCRIPTION
**Pull request by MalikG15**

### CODE REVIEW

File: src/codeu/chat/client/ClientUser.java [hunk 1]
```diff
@@ -84,6 +82,7 @@ public boolean signInUser(String name, String password) {
 	    return (prev != current);
     }
 
+
     System.out.println("Login was UNSUCCESSFUL due to your username and password combination of: " + name + " " + password);
```
Comment (GrayTurtle) [inline src/codeu/chat/client/ClientUser.java]: We should have this go into LOG. And we probably shouldn't be displaying a password lol

File: src/codeu/chat/client/ClientUser.java [hunk 2]
```diff
@@ -171,4 +166,12 @@ public String showUserInfo(String uname) {
   public static void printUser(User user) {
     System.out.println(getUserInfoString(user));
   }
+
+  public int getMessageCount(Uuid userid) {
+      return view.getMessageCount(userid);
+  }
+  
+  public int increaseMessageCount(Uuid userid) {
+	  return controller.updateMessageCount(userid);
```
Comment (GrayTurtle) [inline src/codeu/chat/client/ClientUser.java]: I don't really know if this really matters, but I noticed that our indentions are different. Looks like Google uses spaces and we use tabs.

File: src/codeu/chat/client/View.java [hunk 3]
```diff
@@ -280,4 +280,49 @@ public Uuid getUserGeneration() {
...
+		  else {
+		      LOG.error("Response from server failed.");
+		  }
+	  }
+	  catch (Exception ex) {
+	      System.out.println("ERROR: Exception during call on server. Check log for details.");
+	      LOG.error(ex, "Exception during call on server.");
+	    }
+	  
+  return messageCount;
```
Comment (GrayTurtle) [inline src/codeu/chat/client/View.java]: nit: tab

File: src/codeu/chat/client/ClientUser.java (hunk 1 above)
Comment (MalikG15) [reply src/codeu/chat/client/ClientUser.java]: I couldn't get log to work in this class lol.

**Pull request merged**